examples/chart/teleport-cluster/example-values/full-kubernetes-features.yaml

```
@@ -0,0 +1,80 @@
+# This example demonstrates how to tune Kubernetes scheduling constraints through the chart.
+# The fictional cluster runs on 3 zones and has a dedicated node set for the proxies.
+# Proxy nodes are tainted and labeled with `teleport.dev/role: proxy`.
+# Auth data is stored in a PVC, so it cannot be scaled-up. However, we can run multiple proxy replicas.
+clusterName: teleport.example.com
+
+auth:
+  resources:
+    requests:
+      cpu: "3"
+      memory: "6GiB"
+    limits:
+      cpu: "3"
+      memory: "6GiB"
+
+proxy:
+  resources:
+    requests:
+      cpu: "1"
+      memory: "4GiB"
+    limits:
+      cpu: "1"
+      memory: "4GiB"
```

```suggestion
auth:
  resources:
    requests:
      cpu: "25m"
      memory: "25Mi"
    limits:
      cpu: "4"
      memory: "4Gi"

proxy:
  resources:
    requests:
      cpu: "25m"
      memory: "25Mi"
    limits:
      cpu: "4"
      memory: "4Gi"
```

These match the limits we set by default for Teleport Cloud, so might be a little more useful if people just copy/paste them blindly...

---

Mismatched memory requests and limits 😭